docs/actions-scripts.md

```diff
@@ -34,7 +34,7 @@ Scripted actions can read and write output files that are saved in the workspace
 
 Outputs labelled `highly_sensitive` will not be visible to researchers. This is a [deliberate design feature of OpenSAFELY](https://www.opensafely.org/about/), intended to reduce the risk of disclosure of sensitive information. Outputs should **always** be classed as `highly_sensitive` if they are:
 
-- Pseudonymised patient-level outputs derived from queries run against Level 1 and 2 data, i.e., a specific study dataset generated by a [dataset_definition](https://docs.opensafely.org/ehrql/).
+- Pseudonymised patient-level outputs derived from queries run against Level 1 and 2 data, i.e., a specific study dataset generated by a [dataset definition](ehrql/index.md#a-dataset-definition).
```

Comment:
Oops, it was probably me who used an absolute URL.

But I don't think we should be linking to halfway down the ehrQL index page.  If somebody knows what a dataset definition is, they won't need to follow the link; if they don't, it'd be better to take them to the ehrQL index page.  But this is a judgement call, so feel free to disregard.